src/app/formalVerification.js

```diff
@@ -0,0 +1,30 @@
+var $ = require('jquery');
+
+function FormalVerification (outputElement, renderer) {
+  this.outputElement = outputElement;
+  this.renderer = renderer;
+}
+
+FormalVerification.prototype.compiling = function () {
+  $('#formalVerificationInput', this.outputElement).val('');
+  $('#formalVerificationErrors').empty();
+};
+
+FormalVerification.prototype.compilationFinished = function (compilationResult) {
+  if (compilationResult.formal !== undefined) {
+    if (compilationResult.formal['why3'] !== undefined) {
+      $('#formalVerificationInput', this.outputElement).val(
+        '(* copy this to http://why3.lri.fr/try/ *)' +
```

Comment:
The problem is that I want to keep the textarea small. Its only purpose is to be able to copy it to the clipboard - you can read it after you pasted it into the why3 tool. Because it is small, it will only display the first line of text and that is (due to whatever reason) empty, so I added this prefix here to make it clear _what_ to copy.